doc/source/release-notes/Percona-Server-8.0.19-10.rst

```diff
@@ -0,0 +1,56 @@
+.. _PS-8.0.19-10:
+
+================================================================================
+*Percona Server for MySQL* 8.0.19-10
+================================================================================
+
+:Date: March 11, 2020
```

Comment:
date changed

Reply:
The date is based on the Release Management Dashboard. When the date is updated, the change is propogated to the release note by the Jira-bot. At this point, I'm manually changing the date.